src/main/webapp/src/app/model/garden.model.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Garden interface that corresponds with design doc specs.
+ *
+ */
+export interface Garden {
+  id: string;
+  name: string;
+  biography: string;
+  lat: number;
+  lng: number;
+  zipCode: string;
```

Comment:
If we are using the zip code for matching, how is the lat lng going to be used? Who is using the lat lng and for what?( In terms of us and also thinking about the user)

Reply:
It would be mostly used for Priority Alpha features -- for better nearby gardens searching, etc.